profile/src/main/java/rdx/works/profile/data/repository/HostInfoRepository.kt

```diff
@@ -23,7 +23,7 @@ class HostInfoRepositoryImpl @Inject constructor(
     override suspend fun getHostId(): HostId {
         val sargonOs = sargonOsManager.sargonOs
 
-        return withContext(defaultDispatcher) { sargonOs.resolveHostId() }
+        return withContext(defaultDispatcher) { sargonOs.hostId() }
```

Comment:
I think hostId is no longer suspending.

Reply:
I prefer making the uniffi calls on the background thread to prevent UI performance issues and this requires keeping the method suspending.